File: src/jvm/org/pantsbuild/example/codegen/java/PersonSample.java
package org.pantsbuild.example.codegen.java;

import org.pantsbuild.example.codegen.protos.Person;

public class PersonSample {
  public static void main(String[] args) {
    Person p = Person.newBuilder()
      .setName("Someone")
      .setId(1234)
      .setEmail("someone@example.com")
      .build();
    System.out.println(p);
  }
}

File: src/jvm/org/pantsbuild/example/codegen/scala/PersonSample.scala
package org.pantsbuild.example.codegen.scala

import org.pantsbuild.example.codegen.protos.person.Person

object PersonSample {
  def main(args: Array[String]): Unit = {
    val p = Person(
      name = "Someone",
      id = 1234,
      email = "someone@example.com",
    )
    println(p)
  }
}

